app/customer_routes.py

```diff
@@ -0,0 +1,104 @@
+from flask import Blueprint, jsonify, make_response, request
+
+from app import db
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app.models.video import Video
+from app.rental_routes import deletes_rentals
+
+customers_bp = Blueprint("customers",__name__, url_prefix="/customers")
+
+@customers_bp.route("",methods=["GET"])
+
+def get_customers():
+    customers = Customer.query.all()
+    customers_response = []
+    for customer in customers:
+        customers_response.append(customer.to_dict())
+
+    return jsonify(customers_response),200
+
+@customers_bp.route("/<customer_id>",methods=["GET"])
+def get_customer_by_id(customer_id):
+    
+    try:
+        customer_id = int(customer_id)
+    except ValueError:
+        return make_response(jsonify({"error": "Invalid ID"}), 400)
+    customer = Customer.query.get(customer_id)
+    if not customer:
+        return make_response(jsonify({'message': f'Customer {customer_id} was not found'}),404)
```

Comment:
Just a note, the invalid params code would be great to somehow dry up and abstract out. There are, I believe decorators you can research. 

Not expected as part of the project, just a suggestion for future Flask learning.